src/main/kotlin/step4/racingcar/racingcar/Car.kt

```diff
@@ -0,0 +1,25 @@
+package step4.racingcar.racingcar
+
+class Car(
+    val racerName: String
+) {
+    init {
+        if (racerName.length > RACER_NAME_LENGTH_LIMIT) {
+            throw IllegalArgumentException("입력된 선수의 이름이 5자 이상인 선수가 있습니다.")
+        }
+    }
+
+    private val _movements: MutableList<Boolean> = mutableListOf()
+
+    val movements
+        get() = _movements.toList()
+
+    fun moveOrStop(input: Int) {
+        _movements.add(input > GO_NUMBER)
+    }
+
+    companion object {
+        const val RACER_NAME_LENGTH_LIMIT = 5
```

Comment:
```suggestion
        private const val RACER_NAME_LENGTH_LIMIT = 5
```

RACER_NAME_LENGTH_LIMIT 는 현재 Car 에서만 사용하고 있네요. 이런 경우에는 Car에서만 사용할 수 있도록 접근제어자를 private로 해주세요.